solr/core/src/test/org/apache/solr/uninverting/TestFieldCacheVsDocValues.java

```diff
@@ -122,7 +122,7 @@ public void testSortedSetVariableLengthVsUninvertedField() throws Exception {
   // LUCENE-4853
   public void testHugeBinaryValues() throws Exception {
     Analyzer analyzer = new MockAnalyzer(random());
-    // FSDirectory because SimpleText will consume gobbs of
+    // FSDirectory because SimpleText will consume gobs of
```

Comment:
```suggestion
    // FSDirectory because SimpleText will consume lots of
```

assuming natural language "lots" was meant here rather than technical terms "blobs" or similar?